src/lib/builders/dialog/create.ts

```diff
@@ -256,11 +256,11 @@ export function createDialog(props?: CreateDialogProps) {
 	const portalled = makeElement(name('portalled'), {
 		stores: portal,
 		returned: ($portal) => ({
-			'data-portal': $portal ? '' : undefined,
+			'data-portal': $portal !== null ? '' : undefined,
```

Comment:
We should probably standardize this in an util

Reply:
Good call!